configuration/analyzer_config.json

```diff
@@ -229,6 +229,15 @@
     "python_module": "file_info.FileInfo",
     "soft_time_limit": 30
   },
+  "Floss": {
+    "type": "file",
+    "description": "Static Analysis of Binaries",
+    "python_module": "floss.Floss",
+    "soft_time_limit": 30,
```

Comment:
are you sure this time limit is enough ? did you try to run the binary locally against any file and get an approximation of average time the process takes ? a value like `120` might be more suitable

Reply:
No, I am changing it to 500, Since `capa` also have the same.